docs/cloud/cloud-tables/cloud-table-upload-data.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Upload data
+layout: default
+parent: Manage tables
+grand_parent: Cloud
+nav_order: 2
+---
+
+# How do I upload data to FeatureBase Cloud?
+{: .no_toc }
+
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+* [Create a database](/docs/cloud/cloud-databases/cloud-db-create)
+
+## Naming standards
+
+{% include /concepts/standard-naming-obj.md%}
+{% include /cloud-table/cloud-standard-naming-table.md %}
+
+## Step 1: Navigate to Upload Data
```

Comment:
There are two things the user needs to know BEFORE they start uploading, both in NOTE callouts after step2.

You handle this by adding a Heading 2 here then making subsequent headings H3

```
## Uploading data to a table

{: .important}
Table columns cannot be changed once the table is defined.

### Step 1
...
```